AmplifyPlugins/Geo/AWSLocationGeoPlugin/Configuration/AWSLocationGeoPluginConfiguration.swift

```diff
@@ -53,24 +54,34 @@ public struct AWSLocationGeoPluginConfiguration {
                 throw GeoPluginConfigError.searchDefaultNotFound(indexName: defaultSearchIndex)
             }
         }
+        
+        let trackerConfigJSON = configObject[Section.tracker.key]
```

Comment:
```
"trackers": { 
       "items": ["tracker823xyz", "tracker725cba"],
       "default": "tracker823xyz"
} 
```
i saw this in the approved CLI contract. do we need to store default and other trackers separately?

Reply:
Good point, I've updated the Geo configuration to match the cli contract.